internal/scan-manager/scanners/veracode/pom.xml

```diff
@@ -0,0 +1,78 @@
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.wso2.security.tools</groupId>
+        <artifactId>scan-manager-parent</artifactId>
+        <version>1.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.wso2.security.tools.scanmanager</groupId>
+    <artifactId>scanners-veracode</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <name>Scan Manager - Scanner - Veracode</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.veracode.vosp.api.wrappers</groupId>
+            <artifactId>vosp-api-wrappers-java</artifactId>
+            <version>${vosp.api.wrappers.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>${log4j.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.security.tools.scanmanager</groupId>
+            <artifactId>scan-manager-scanners-common</artifactId>
+            <version>${org.wso2.security.tools.scanmanager.scanner.common.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.security.tools.scanmanager</groupId>
+            <artifactId>scan-manager-common-internal</artifactId>
+            <version>${org.wso2.security.tools.scanmanager.common.internal.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.security.tools.scanmanager</groupId>
+            <artifactId>scan-manager-common</artifactId>
+            <version>${org.wso2.security.tools.scanmanager.common.version}</version>
+        </dependency>
+    </dependencies>
+    <properties>
+        <vosp.api.wrappers.version>18.6.5.3</vosp.api.wrappers.version>
+        <org.springframework.boot.starter.version>1.5.6.RELEASE</org.springframework.boot.starter.version>
+        <log4j.version>1.2.17</log4j.version>
+        <org.wso2.security.tools.scanmanager.common.internal.version>1.0-SNAPSHOT
+        </org.wso2.security.tools.scanmanager.common.internal.version>
```

Comment:
Let's move the ending tag to the same line as starting tag. It's ok to exceed the char limit here.